ui/src/components/field/QasField.vue

```diff
@@ -159,11 +159,17 @@ export default {
     // This computed will change the key name when the server sends different key.
     formattedField () {
       const field = {}
+      const { label, required, type } = this.field
```

Comment:
O mais correto não seria pegar o valor do `field` que vai ser gerado ao invés do `this.fields`, no caso após o "for".

Reply:
feito